### PR TITLE
Browserstack upload 0.1.2

### PR DESCRIPTION
**Pull request by marcelkalveram**
TagCheck

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [X] __I will not move an already shared step version's tag to another commit__
- [X] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [X] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [X] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [X] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/browserstack-upload/0.1.2/step.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+project_type_tags:
+- ios
+- android
+- xamarin
+- react-native
+- cordova
+- ionic
+type_tags:
+- utility
+- test
```
Comment (BirmacherAkos) [inline steps/browserstack-upload/0.1.2/step.yml]: I think the test type_tag would be enough. I don't see why we should use the utility tpye_tag for this step.

File: steps/browserstack-upload/0.1.2/step.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
```
Comment (BirmacherAkos) [inline steps/browserstack-upload/0.1.2/step.yml]: Should this step run only on the OSX stacks?
What about the Ubuntu (Android & Docker on Ubuntu) ones? 

If you want to use this step on Ubuntu as well please add the `apt-get` dependency list just like you did with the `brew`.

File: steps/browserstack-upload/0.1.2/step.yml (hunk 1 above)
Comment (marcelkalveram) [reply steps/browserstack-upload/0.1.2/step.yml]: Thanks for the feedback, @BirmacherAkos ! I'll update this.

**Comment by BirmacherAkos** [comment]
Hi @marcelkalveram !

Thanks for this awesome step.  🥇 
I really like the idea to have a step for Browserstack.

Could you please check my comments?

File: steps/browserstack-upload/0.1.2/step.yml (hunk 2 above)
Comment (marcelkalveram) [reply steps/browserstack-upload/0.1.2/step.yml]: @BirmacherAkos I haven't tested the other stacks. I guess IPA files won't build there anyway.
For the APKs though, I can give it a try and amend the step accordingly. Give me a day or two.

**Comment by BirmacherAkos** [comment]
Hi @marcelkalveram !

I have merged the https://github.com/bitrise-io/bitrise-steplib/pull/1766 PR.
So I will close this PR.

**Pull request closed without merging**